chart/crds/vaultdynamicsecret-crd.yaml

```
@@ -74,6 +75,18 @@ spec:
               namespace:
                 description: Namespace where the secrets engine is mounted in Vault.
                 type: string
+              params:
+                additionalProperties:
+                  type: string
+                description: Params that can be passed when requesting credentials/secrets.
+                  Please consult https://developer.hashicorp.com/vault/docs/secrets
+                  if one is uncertain about what 'params' should/can be set to.
```

Our documentation style should address the reader directly as "you" (and same comment for path and the other places this doc string crops up)

```suggestion
                  if you are uncertain about what 'params' should/can be set to.
```

---

Fixed in https://github.com/hashicorp/vault-secrets-operator/commit/500f1e0f926cac8106c76751f8316a083cf65d90